src/storage/sftp.py

```diff
@@ -69,12 +72,12 @@ def _connect():
     # Start SSH session based on authentication type
     try:
         if AUTHENTICATION_TYPE == "password":
-            # FIXME: Don't save password in unencrypted config file
+            # FIXME: Use encrypted KVStore for password
             password = CONFIG.get(_name, "password")
             transport.connect(username=USERNAME, password=password)
 
         elif AUTHENTICATION_TYPE == "key":
-            # FIXME: Don't save key passphrase in unencrypted config file
+            # FIXME: Use encrypted KVStore for password
             key_filepath = CONFIG.get(_name, "key_filepath")
             key_passphrase = CONFIG.get(_name, "key_passphrase")
```

Comment:
We cannot use the kvstore here yet because there is no way to prompt the user for a password (we need to decide on a gui toolkit by then). I guess this should be implemented when working on issue #22.